test/unit/event/send-submission-transfer-event.test.js

```diff
@@ -0,0 +1,141 @@
+const mockSendEvents = jest.fn()
+const mockPublishEvent = jest.fn()
+const MockPublishEventBatch = jest.fn().mockImplementation(() => {
+  return {
+    sendEvents: mockSendEvents
+  }
+})
+const MockEventPublisher = jest.fn().mockImplementation(() => {
+  return {
+    publishEvent: mockPublishEvent
+  }
+})
+jest.mock('ffc-pay-event-publisher', () => {
+  return {
+    PublishEventBatch: MockPublishEventBatch,
+    EventPublisher: MockEventPublisher
+  }
+})
+jest.mock('../../../app/config')
+const config = require('../../../app/config')
+const { BATCH_CREATED } = require('../../../app/constants/events')
+const { SOURCE } = require('../../../app/constants/source')
+const sendSubmissionTransferEvents = require('../../../app/event/send-submission-transfer-event')
+
+let batch
+let paymentRequest
+let filename
+
+beforeEach(() => {
+  paymentRequest = JSON.parse(JSON.stringify(require('../../mocks/payment-request')))
+  batch = JSON.parse(JSON.stringify(require('../../mocks/batch')))
+  batch.paymentRequests = [paymentRequest, paymentRequest]
+
+  filename = 'test.csv'
```

Comment:
See my comments here: https://github.com/DEFRA/ffc-pay-submission/pull/46/files#r1133988168

Reply:
I think is different as the content of batch is subjective to this test suit.

Reply:
Sorry, what do you mean?

Reply:
Apologies, I misinterpreted what you'd said.

You're right, I've updated as suggested.

Reply:
You're grand!